### PR TITLE
Src 5445 payload

### PR DESCRIPTION
**Pull request by biofotis**
## Proposed changes

Please explain the changes you made. Add pictures or videos to explain them better if appropiate.

## Types of changes

What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

Put an x in the boxes that apply. You can also fill these out after creating the PR. This is a reminder of what we should look for before merging this code. I have:

- [ ] Read and follow the [contributing guidelines](https://github.com/shadow-robot/sr_documentation/blob/master/CONTRIBUTING.md).
- [ ] Checked that all tests pass with my changes
- [ ] Added tests (automatic or manual) that prove the fix is effective or that the feature works
- [ ] Added necessary documentation (if appropriate)
- [ ] Added the corresponding license to each file and add the current year of any one that you modified.
- [ ] Tested on real hardware (if appropriate)
￼

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: sr_robot_launch/launch/sr_hardware_control_loop.launch [hunk 1]
```diff
@@ -51,10 +51,10 @@
   <!-- Set to true to spawn the position controllers for the arm-->
   <arg name="arm_position" default="$(eval not arm_trajectory)"/>
   <arg name="arm_speed_scale" default="0.5"/>
-  <arg if="$(arg hand_ctrl)" name="arm_payload_mass" default="5.00"/>
-  <arg if="$(arg hand_ctrl)" name="arm_payload_cog" default="[0.0, 0.0, 0.12]"/>
-  <arg unless="$(arg hand_ctrl)" name="arm_payload_mass" default="0.0"/>
-  <arg unless="$(arg hand_ctrl)" name="arm_payload_cog" default="[0.0, 0.0, 0.0]"/>
+  <arg if="$(arg hand)" name="arm_payload_mass" default="5.00"/>
```
Comment (mikramarc) [inline sr_robot_launch/launch/sr_hardware_control_loop.launch:54]: you're changing this but seems like other launchfiles still use hand_ctrl arg callling this file, right?
https://github.com/shadow-robot/sr_interface/blob/296ae6946875de85c3b02fe4679dc7116476d1d6/sr_robot_launch/launch/sr_ur_arm_hand.launch#L56
Have you tested this PR?

**Review by mikramarc — CHANGES_REQUESTED** [review]

File: sr_robot_launch/launch/sr_bimanual_ur10arms_hands.launch [hunk 2]
```diff
@@ -55,6 +55,8 @@
   <!-- The control mode PWM (true) or torque (false) -->
   <!-- Set to true by default for now as torque control is not available yet -->
   <arg name="pwm_control" default="true"/>
+  <!-- Set the hand type here: hand_e_plus, hand_e, Hand_lite, hand_extra_lite -->
```
Comment (carebare47) [inline sr_robot_launch/launch/sr_bimanual_ur10arms_hands.launch]: we should make sure these examples are all lower-case, unless we plan on somehow adding support for mixed-case strings

File: sr_robot_launch/launch/sr_left_ur10arm_hand.launch [hunk 3]
```diff
@@ -44,6 +44,8 @@
   <arg name="pwm_control" default="true"/>
   <!-- Set to true if you want to use grasp controller -->
   <arg name="grasp_controller" default="false"/>
+  <!-- Set the hand type here: hand_e_plus, hand_e, Hand_lite, hand_extra_lite -->
```
Comment (carebare47) [inline sr_robot_launch/launch/sr_left_ur10arm_hand.launch]: same here

File: sr_robot_launch/launch/sr_left_ur5arm_hand.launch [hunk 4]
```diff
@@ -44,6 +44,8 @@
   <arg name="pwm_control" default="true"/>
   <!-- Set to true if you want to use grasp controller -->
   <arg name="grasp_controller" default="false"/>
+  <!-- Set the hand type here: hand_e_plus, hand_e, Hand_lite, hand_extra_lite -->
```
Comment (carebare47) [inline sr_robot_launch/launch/sr_left_ur5arm_hand.launch]: and here

File: sr_robot_launch/launch/sr_right_ur10arm_hand.launch [hunk 5]
```diff
@@ -44,6 +44,8 @@
   <arg name="pwm_control" default="true"/>
   <!-- Set to true if you want to use grasp controller -->
   <arg name="grasp_controller" default="false"/>
+  <!-- Set the hand type here: hand_e_plus, hand_e, Hand_lite, hand_extra_lite -->
```
Comment (carebare47) [inline sr_robot_launch/launch/sr_right_ur10arm_hand.launch]: same

File: sr_robot_launch/launch/sr_right_ur5arm_hand.launch [hunk 6]
```diff
@@ -44,6 +44,8 @@
   <arg name="pwm_control" default="true"/>
   <!-- Set to true if you want to use grasp controller -->
   <arg name="grasp_controller" default="false"/>
+  <!-- Set the hand type here: hand_e_plus, hand_e, Hand_lite, hand_extra_lite -->
```
Comment (carebare47) [inline sr_robot_launch/launch/sr_right_ur5arm_hand.launch]: and here

File: sr_robot_launch/launch/sr_hardware_control_loop.launch (hunk 1 above)
Comment (biofotis) [reply sr_robot_launch/launch/sr_hardware_control_loop.launch:54]: Yes it was tested. The file you linked pass the hand_ctrl to load or not the controllers. In the low level hardware control loop we only want to set the payload if hand is there and not hand_ctrl as for the arm would be the same regardless if we load the controllers for hand.

**Review by mikramarc — APPROVED** [review]

**Pull request merged**